src/detail-page/components/TabJobContent/FetchJobContent.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, CircularProgress, Input } from '@material-ui/core';
+import styled from 'styled-components';
+import * as React from 'react';
```

Comment:
There's no need for the `*` - we can simply write:
```
import React from 'react';
```
We can also add named imports for `useState` and `useCallback`, that will allow us to remove the repeated `React` prefix 🙃
```
import React, { useState, useCallback } from 'react';